airbyte-cdk/python/airbyte_cdk/sources/file_based/config/csv_format.py

```diff
@@ -81,6 +86,11 @@ class Config:
         default=DEFAULT_FALSE_VALUES,
         description="A set of case-sensitive strings that should be interpreted as false values.",
     )
+    inference_type: InferenceType = Field(
```

Comment:
do we want to completely hide the option using `airbyte_hidden` to prevent new sources from using inference?

Reply:
I'd like to.